migtests/tests/pg/assessment-report-test/expectedAssessmentReport.json

```diff
@@ -2182,9 +2198,26 @@
 			"Query": "SELECT *\nFROM xmltable(\n    $1\n    PASSING $2\n    COLUMNS \n        name TEXT PATH $3\n)",
 			"DocsLink": "https://docs.yugabyte.com/preview/yugabyte-voyager/known-issues/postgresql/#xml-functions-is-not-yet-supported",
 			"MinimumVersionsFixedIn": null
+		},
+		{
+			"ConstructTypeName": "Large Object Functions",
+			"Query": "SELECT lo_create($1)",
+			"DocsLink": "",
+			"MinimumVersionsFixedIn": null
 		}
 	],
 	"UnsupportedPlPgSqlObjects": [
+		{
+			"FeatureName": "Large Object Functions",
+			"Objects": [
+				{
+					"ObjectType": "FUNCTION",
+					"ObjectName": "public.manage_large_object",
+					"SqlStatement": "SELECT lo_unlink(loid);"
```

Comment:
Q: Why is this coming with `SELECT` here? In the DDL below i see `PERFORM`

Reply:
`PERFORM` is internally treated as `SELECT ` in db so its parse tree is giving the query as SELECT.